### PR TITLE
New article "Windowed vs. visual hosting of WebView2"

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/windowed-vs-visual-hosting?branch=pr-en-us-2205

This PR is based off latest main (not based on PR 2041).  This PR requests merging into main (not into PR 2041).
This PR manually copied the changes from PR 2041, and replaces [PR 2041](https://github.com/MicrosoftDocs/edge-developer/pull/2041).  For commit history, including dan-wesley's contributions, see 2041.

This forked PR enables the contributors to push commits into it, and enables merging directly to origin's main branch.

### CODE REVIEW

File: microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md [hunk 1]
```diff
@@ -0,0 +1,434 @@
+---
+title: Windowed vs. visual hosting of WebView2
+description: Deciding whether to have your app use windowed hosting vs. visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 08/24/2022
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md:9]: @victorthoang - are you able to update date & push a commit into this forked PR?

Comment (victorthoang) [reply microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md]: Responded to you on Teams, but I was able to commit and tested by changing the date.

**Comment by mikehoffms** [comment]
LGTM

**Review by victorthoang — APPROVED** [review]
final reference links added. Appreciate the help and approving.

**Review by JasonAndrewWriter — APPROVED** [review]

**Pull request merged**